### ISSUE TITLE
Add support for custom handlebars helpers

### ISSUE THREAD

**Issue by mrjones2014** [mrjones2014/yaclt#119]
**Is your feature request related to a problem? Please describe.**
There are currently a few built-in handlebars helpers. We should support registering user-defined handlebars helpers.

**Describe the solution you'd like**
I am not sure where these should be defined. `yacltrc.js` is the obvious solution but I am not sure this is the right place. Especially because this makes using other config formats (yml, json) and having custom handlebars helpers mutually exclusive.

**Additional context**
Maybe they should just live in a conventionally named file like `yacltrc.handlebars.js` or `yaclt-handlebars.js` or something. Not sure the best approach to take here.

**Comment by mrjones2014** [comment]
@hsheikhali1 thoughts if you have time

**Comment by hsheikhali1** [comment]
> @hsheikhali1 thoughts if you have time

Yeah this is a tricky one.. part of me wants the ability to define a `.handlebars.js` file and yactl will pick it up .. but part of me feels like there is probably a less intrusive way to do this ?

I'll create a feature branch to experiment not sure how this will work tho :/

**Comment by hsheikhali1** [comment]
Where are the built-in handlebars helpers anyway ? Are you just using handlebars `.registerHelper()` where you need it?

**Comment by mrjones2014** [comment]
They're registered in `cli/index.ts` (https://github.com/mrjones2014/yaclt/blob/master/src/cli/index.ts#L20) and defined in `utils/handlebars-helpers.ts` (https://github.com/mrjones2014/yaclt/blob/master/src/utils/handlebars-helpers.ts)

**Comment by hsheikhali1** [comment]
Going to work on this later today -- have not forgotten about it :)
